### PR TITLE
Implement datamodel

### PR DESCRIPTION
**Pull request by CharlesDLandau**
- Added `components/CaseTable.js` which manages the dominant view (using React useContext)
- Added layout cruft to `components/bodyview.js`
- Added `contexts/casecontroller.js` responsible for global app state (!!) including pre-initialized context provider
- Added Material-UI helper components (e.g. `CssBaseline`) per best practices
- Moved SSoT JSON files to `static` for now
- ~~TODO: reinstate nav.js~~
- TODO: improve linting workflow

### CODE REVIEW

**Comment by CharlesDLandau** [comment]
I suspect that CI is failing because I renamed files in `components` to adopt upper camel. Regardless I need some code review.

**Comment by CharlesDLandau** [comment]
Obviously didn't work...

File: components/CaseTable.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          })}
+        </div>
+
+        <CardActionArea
+          onClick={() => {
+            console.log(value);
+            value.pushCharge();
+          }}
+        >
+          <Typography variant="h6">+ New Case</Typography>
```
Comment (haydenbetts) [inline components/CaseTable.js]: Should this be New Charge?

Comment (CharlesDLandau) [reply components/CaseTable.js]: Yes

**Pull request merged**